Update gateways info (#1531)
ipfs/ipfs-docs: docs/how-to/gateway-troubleshooting.md

---
title: Troubleshooting 
description: Learn how to troubleshoot common issues with IPFS HTTP Gateways
---

# Troubleshooting HTTP Gateways

> IPFS HTTP Gateways (GWs) provide an HTTP-based service that allows IPFS-incompatible browsers, tools and software to access IPFS content. When using HTTP GWs , developers may need to troubleshoot issues like a:

Reviewer: ```suggestion
IPFS HTTP Gateways are an HTTP-based service allowing browsers, tools and software to retrieve IPFS content with HTTP. When using HTTP Gateways, developers may need to troubleshoot issues like a:
```

Let's stick to Gateways. I don't think we need to shorten it.
